SPECS/ig/ig.signatures.json

```
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "ig-0.25.0-govendor-v1.tar.gz": "ba1c9153ee67cd9d1ebd31021ca3848c1099979c229b26bea6ad05470ac486ac",
-  "ig-0.25.0.tar.gz": "3d283e0f6e758b36e291e483e79e7a08c4ce2965a6c11f5c1ab671ced7015a6e"
+  "ig-0.26.0-govendor-v1.tar.gz": "260c4d8f9864cc3dd54567a854c5ab8dcb0c8246cf1caa1d3cb54c8e25328b90",
```

```suggestion
  "ig-0.26.0-govendor-v1.tar.gz": "79bd7c40cb29f4eb7e8f002f9ad8bc30928c48bea2ea6766e73d18c36513b775",
```
This was the hash when I generated the vendored go. Checking if the difference is expected.

```bash
bash generate_source_tarball.sh --srcTarball ig-0.26.0.tar.gz --pkgVersion 0.26.0
```

---

I think I should create one locally to see if there is a correlation with respect to environment.

---

```bash
mfrw in 🌐 curie in ~/ma/3.0/SPECS/ig on  francis/bump-ig:git@github.com:eiffel-fl/CBL-Mariner.git/francis/bump-ig [$?] 
fsh ❯ bash generate_source_tarball.sh --srcTarball ig-0.26.0.tar.gz --pkgVersion 0.26.0
--srcTarball      -> ig-0.26.0.tar.gz
--outFolder       -> /home/mfrw/mariner-org/3.0/SPECS/ig
--pkgVersion      -> 0.26.0
--vendorVersion   -> 1
-- create temp folder
Unpacking source tarball...
Vendor go modules...

=========================
Tar vendored tarball
ig vendored modules are available at /home/mfrw/mariner-org/3.0/SPECS/ig/ig-0.26.0-govendor-v1.tar.gz
+++ cleanup -> remove /tmp/tmp.mo0Q0In2KL
⏎                                                                                                         

mfrw in 🌐 curie in ~/ma/3.0/SPECS/ig on  francis/bump-ig:git@github.com:eiffel-fl/CBL-Mariner.git/francis/bump-ig [$!?] 
fsh ❯ ls
generate_source_tarball.sh  ig-0.26.0-govendor-v1.tar.gz  ig-0.26.0.tar.gz  ig.signatures.json  ig.spec
mfrw in 🌐 curie in ~/ma/3.0/SPECS/ig on  francis/bump-ig:git@github.com:eiffel-fl/CBL-Mariner.git/francis/bump-ig [$!?] 
fsh ❯ cat ig.signatures.json 
{
 "Signatures": {
  "ig-0.26.0-govendor-v1.tar.gz": "260c4d8f9864cc3dd54567a854c5ab8dcb0c8246cf1caa1d3cb54c8e25328b90",
  "ig-0.26.0.tar.gz": "22268a32daccacfc01afdb66942c75ba0394d47c9df4f19720968dd6a8ad758d"
 }
}⏎

mfrw in 🌐 curie in ~/ma/3.0/SPECS/ig on  francis/bump-ig:git@github.com:eiffel-fl/CBL-Mariner.git/francis/bump-ig [$!?] 
fsh ❯ go version
go version go1.22.3 linux/amd64
```

---

@christopherco - I will share the vendor tar ball to you.
As we discussed offline, it was indeed go `1.21` vs `1.22` :)